code/modules/reagents/reagent_containers/food/snacks/baked_goods.dm

```diff
@@ -0,0 +1,231 @@
+/////////////////
+// Baked Goods //
+/////////////////
+
+/obj/item/chems/food/snacks/muffin
+	name = "muffin"
+	desc = "A delicious and spongy little cake."
+	icon_state = "muffin"
+	filling_color = "#e0cf9b"
+	center_of_mass = @"{'x':17,'y':4}"
+	nutriment_desc = list("sweetness" = 3, "muffin" = 3)
+	nutriment_amt = 6
+	bitesize = 2
+
+/obj/item/chems/food/snacks/bananapie
+	name = "banana cream pie"
+	desc = "Just like back home, on clown planet! HONK!"
+	icon_state = "pie"
+	trash = /obj/item/trash/plate
+	filling_color = "#fbffb8"
+	center_of_mass = @"{'x':16,'y':13}"
+	nutriment_desc = list("pie" = 3, "cream" = 2)
+	nutriment_amt = 4
+	bitesize = 3
+/obj/item/chems/food/snacks/pie/Initialize()
```

Comment:
There are some missing newlines; not going to comment on every one.

Reply:
*relatively* sure that these existed in the already-existing similarly-grouped runs that I copied from `snacks.dm`, but I can go through and fix these if you'd like.

Reply:
Yeah, the old formatting is bad (see all the `.=..()` in Initialize) but having the whitespace be normal at least would be nice.